src/server/webpack-config.js

```diff
@@ -83,6 +83,7 @@ export default (options = {}) => {
       loaders: [
         babelLoader(/\.js$/, isRelayEnabled),
         babelLoader(/\.jsx$/, isRelayEnabled),
+        { test: /\.css$/, loader: 'style-loader!css-loader' },
```

Comment:
The shorthand is `style!css`, you don't need to include the loader part, but having it there is also fine